src/transformers/feature_extraction_utils.py

```diff
@@ -431,16 +433,23 @@ def from_dict(cls, feature_extractor_dict: Dict[str, Any], **kwargs) -> PreTrain
         else:
             return feature_extractor
 
-    def to_dict(self) -> Dict[str, Any]:
+    def to_dict(self, processor_class_str: Optional[str] = None) -> Dict[str, Any]:
         """
         Serializes this instance to a Python dictionary.
 
+        Args:
+            processor_class_str (:obj:`str`, `optional`):
+                Processor class to be saved in config.
```

Comment:
```suggestion
        Args:
            processor_class_str (`str`, *optional*):
                Processor class to be saved in config.
```
Docstrings are in Markdown now.